code/game/objects/items/weapons/blades.dm

```diff
@@ -136,22 +136,29 @@
 
 	switch(loaded_reagent.type)
 		if(/datum/reagent/medicine/tramadol)
-			M.apply_status_effect(/datum/status_effect/incapacitating/harvester_slowdown, 2 SECONDS)
+			M.apply_damage(force*0.6, BRUTE, user.zone_selected)
+			M.apply_status_effect(/datum/status_effect/incapacitating/harvester_slowdown, 1 SECONDS)
 
 		if(/datum/reagent/medicine/kelotane)
-			var/turf/target = get_turf(M)
-			target.ignite()
 			var/list/cone_turfs = generate_cone(user, 2, 1, 91, Get_Angle(user, M.loc))
 			for(var/X in cone_turfs)
 				var/turf/T = X
-				T.ignite()
+				for(var/mob/living/C in T)
+					C.flamer_fire_act(10)
+					C.apply_damage(20, BURN, user.zone_selected, C.get_soft_armor("fire", user.zone_selected))
+					if(C.IgniteMob())
+						C.visible_message("<span class='danger'>[C] bursts into flames!</span>","[isxeno(C)?"<span class='xenodanger'>":"<span class='highdanger'>"]You burst into flames!</span>")
```

Comment:
Why are you using C for a living mob use better var names

Reply:
changed